substrate/bin/node/cli/tests/runtime_api_works.rs

```diff
@@ -0,0 +1,56 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#![cfg(unix)]
+use codec::Decode;
+use polkadot_sdk::sp_runtime::traits::NumberFor;
+use std::time::Duration;
+use substrate_cli_test_utils as common;
+use substrate_rpc_client::{ws_client, StateApi};
+
+#[tokio::test]
+async fn transaction_storage_runtime_api_call_works() {
```

Comment:
I don't see that much value in this test :D

Reply:
> I don't see that much value in this test :D

Yeah, really not neede, I just didn't find any example for substrate-node.
Removed here: https://github.com/paritytech/polkadot-sdk/pull/10656/commits/d0e55e8bc676444f8142424c0091414194d081de